spec/service/systemd_service_spec.rb

```diff
@@ -116,4 +116,29 @@
       expect(@service.status).to eq(status)
     end
   end
+
+  describe "#show" do
+
+    it "returns a hash of runtime information" do
+      output = <<-EOS
+MainPID=29189
+ExecStart={ path=/bin/sh ; argv[]=/bin/sh -c /bin/evmserver.sh start ; status=0/0 }
+ExecStop={ path=/bin/sh ; argv[]=/bin/sh -c /bin/evmserver.sh stop ; status=0/0 }
+ControlGroup=/system.slice/evmserverd.service
+MemoryCurrent=2865373184
+EOS
+
+      hash = {
+        "MainPID"       => "29189",
+        "ExecStart"     => "{ path=/bin/sh ; argv[]=/bin/sh -c /bin/evmserver.sh start ; status=0/0 }",
+        "ExecStop"      => "{ path=/bin/sh ; argv[]=/bin/sh -c /bin/evmserver.sh stop ; status=0/0 }",
+        "ControlGroup"  => "/system.slice/evmserverd.service",
+        "MemoryCurrent" => "2865373184"
+      }
+      expect(LinuxAdmin::Common).to receive(:run!)
+        .with(LinuxAdmin::Common.cmd(:systemctl),
```

Comment:
Would it make sense to put the command args in another variable to improve readability by allowing the `expect` to be written on one line?